Connectors/Revit/Speckle.Connectors.RevitShared/ElementIdHelper.cs

```diff
@@ -5,13 +5,14 @@ namespace Speckle.Connectors.RevitShared;
 
 public static class ElementIdHelper
 {
-  public static ElementId Parse(string idStr)
+  public static ElementId GetElementIdByUniqueId(Document doc, string uniqueId)
```

Comment:
I'd name this `GetElementIdFromUniqueId`, as we're transmuting an unique id into an element id, and in my head sounds clearer